build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 
 version '0.1'
 
-sourceCompatibility = 1.8
+sourceCompatibility = 1.11
```

Comment:
Nice :). That will make it impossible to run on JDK 8 by the way. But that's fine